includes/admin/reporting/reports.php

```diff
@@ -2545,377 +2544,454 @@ function display_export_report() {
-                            </form>
-                        </div>
-                    </div>
-
-                    <div class="postbox edd-export-download-history">
-                        <h3 class="hndle"><span><?php esc_html_e('Export File Download Logs','easy-digital-downloads' ); ?></span></h3>
-                        <div class="inside">
-                            <p><?php esc_html_e( 'Download a CSV of file download logs.', 'easy-digital-downloads' ); ?></p>
-                            <form id="edd-export-file-downloads" class="edd-export-form edd-import-export-form" method="post">
-								<?php echo EDD()->html->product_dropdown(
+							</div>
+							<?php
+							wp_nonce_field( 'edd_ajax_export', 'edd_ajax_export' );
+
+							?>
+							<input type="hidden" name="edd-export-class" value="EDD_Batch_API_Requests_Export"/>
+							<button type="submit" class="button button-secondary"><?php esc_html_e( 'Generate CSV', 'easy-digital-downloads' ); ?></button>
+						</form>
+					</div>
+				</div>
+
+				<div class="postbox edd-export-download-history">
+					<h2 class="hndle"><span><?php esc_html_e( 'Export File Download Logs', 'easy-digital-downloads' ); ?></span></h2>
+					<div class="inside">
+						<p><?php esc_html_e( 'Download a CSV of file download logs.', 'easy-digital-downloads' ); ?></p>
+						<form id="edd-export-file-downloads" class="edd-export-form edd-import-export-form" method="post">
+						<label for="edd_file_download_export_download" class="screen-reader-text"><?php esc_html_e( 'All Downloads or Select Download', 'easy-digital-downloads' ); ?></label>
+							<?php echo EDD()->html->product_dropdown(
+								array(
+									'name'        => 'download_id',
+									'id'          => 'edd_file_download_export_download',
+									'chosen'      => true,
+									/* translators: the plural post type label */
+									'placeholder' => sprintf( __( 'All %s', 'easy-digital-downloads' ), edd_get_label_plural() ),
+								)
+							);
+							?>
+							<div class="edd-from-to-wrapper">
+								<label for="edd-file-download-export-start" class="screen-reader-text"><?php esc_html_e( 'Export File Download Logs Start Date', 'easy-digital-downloads' ); ?></label>
+								<?php
+								echo EDD()->html->date_field(
 									array(
-										'name'        => 'download_id',
-										'id'          => 'edd_file_download_export_download',
-										'chosen'      => true,
-										/* translators: the plural post type label */
-										'placeholder' => sprintf( __( 'All %s', 'easy-digital-downloads' ), edd_get_label_plural() ),
-									)
-								); ?>
-								<div class="edd-from-to-wrapper">
-									<?php
-
-									echo EDD()->html->date_field( array(
-										'id'          => 'edd-file-download-export-start',
+										'id'          => 'edd_file_download_export_start',
```

Comment:
@LisaCee I think this needs to be switched back to hypens so the `label` is linked properly.